how-to/develop-with-buildd.rst

```diff
@@ -268,3 +292,88 @@ The buildd that you have just installed needs registering with Launchpad so that
     - Non-virtualized builders are much simpler: ``launchpad-buildd`` is cleaned synchronously over XML-RPC at the end of each build, and that's it.  If you use this, then you must be careful not to run any untrusted code on the builder (since a ``chroot`` or container escape could compromise the builder), and you'll need to uncheck "Require virtualized builders" on any PPAs, live file systems, recipes, etc. that you want to be allowed to build on this builder.
 
  4. After 30 seconds or so, the status of the builder on the builders page should be 'Idle'. This page does not auto-update, so refresh!
+
+Running a build on qastaging
+----------------------------
+
+We can use XML-RPC to interact also with qastaging/staging builders. 
+First of all, we should be able to ssh into our bastion and ssh into the ``launchpad-buildd-manager`` unit since it's the one that has the firewall rules to talk with builders.
```

Comment:
Maybe be a little bit more specific for a new comer to understand that you'd need to `sudo -iu stg-launchpad` then `in-model=qastaging juju ssh...` type of deal